ravenframework/SupervisedLearning/ScikitLearn/SVM/SVR.py

```diff
@@ -101,6 +99,10 @@ class cls.
                                                  descr=r"""Enable verbose output. Note that this setting takes advantage
                                                  of a per-process runtime setting in libsvm that, if enabled, may not
                                                  work properly in a multithreaded context.""", default=False))
+    specs.addSub(InputData.parameterInputFactory("normalizeTargets", contentType=InputTypes.BoolType,
+                                                 descr=r"""enables target normalization by centering (subtracting the mean) and dividing by the standard deviation.
+                                                 This is known to make the ROM less sensitive to parameters such as epsilon, gamma, etc.""", default=False))
```

Comment:
I think this can be moved to ScikitLearnBase class

Reply:
@wangcj05, the ScikitLearnBase doesn't have getInputSpecs. Where can I add that?

Reply:
I guess we can put it in the SupervisedLearning class.

Reply:
Yes, That's what I have done. Thanks